### PR TITLE
chore(Tabs): converted examples to TypeScript

### LINKED ISSUE #8297: Tabs: convert examples to TypeScript

### PR DESCRIPTION
**Pull request by kev-kim**
**What**: Closes #8297 

**Additional issues**:

### CODE REVIEW

File: packages/react-core/src/components/Tabs/examples/TabsDefault.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const [isBox, setIsBox] = React.useState<boolean>(false);
+  // Toggle currently active tab
+  const handleTabClick = (
+    event: React.MouseEvent<any> | React.KeyboardEvent | MouseEvent,
+    tabIndex: string | number
+  ) => {
+    setActiveTabKey(tabIndex);
+  };
+
+  const toggleBox = checked => {
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Tabs/examples/TabsDefault.tsx:15]: This param just needs to be typed.

File: packages/react-core/src/components/Tabs/examples/TabsVertical.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  const [isBox, setIsBox] = React.useState<boolean>(false);
+  // Toggle currently active tab
+  const handleTabClick = (
+    event: React.MouseEvent<any> | React.KeyboardEvent | MouseEvent,
+    tabIndex: string | number
+  ) => {
+    setActiveTabKey(tabIndex);
+  };
+
+  const toggleBox = checked => {
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Tabs/examples/TabsVertical.tsx:15]: Typed here as well

File: packages/react-core/src/components/Tabs/examples/TabsVerticalExpandableUncontrolled.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+export const TabsVerticalExpandableUncontrolled: React.FunctionComponent = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState<string | number>(0);
+  // Toggle currently active tab
+  const handleTabClick = (
+    event: React.MouseEvent<any> | React.KeyboardEvent | MouseEvent,
+    tabIndex: string | number
+  ) => {
+    setActiveTabKey(tabIndex);
+  };
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Tabs/examples/TabsVerticalExpandableUncontrolled.tsx:11]: @tlabaj should this state and handleTabClick both be removed to match the other Uncontrolled example?

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: packages/react-core/src/components/Tabs/examples/TabsVerticalExpandableUncontrolled.tsx (hunk 3 above)
Comment (kev-kim) [reply packages/react-core/src/components/Tabs/examples/TabsVerticalExpandableUncontrolled.tsx:11]: @thatblindgeye @tlabaj Pinging so this issue doesn't get lost!

**Review by thatblindgeye — APPROVED** [review]

**Review by tlabaj — APPROVED** [review]
LGTM

**Pull request merged**